ArduPlane/quadplane.cpp

```diff
@@ -624,11 +624,11 @@ bool QuadPlane::setup(void)
         }
         frame_class.set_and_save(new_value);
     }
+    gcs().send_text(MAV_SEVERITY_INFO, "QuadPlane initialise, class: %d, type: %d", frame_class.get(), frame_type.get());
```

Comment:
Could we make this work like Copter's https://github.com/ardupilot/ardupilot/blob/master/ArduCopter/GCS_Mavlink.cpp#L532 ?

Reply:
That would be an improvement; too bad it's so awkward to do in C.

Reply:
Ah, darn.  I thought that stuff was a method on the Motors class by now.  Move on...